pkg/skaffold/deploy/kustomize.go

```diff
@@ -366,8 +382,16 @@ func pathExistsLocally(filename string, workingDir string) (bool, os.FileMode) {
 func (k *KustomizeDeployer) readManifests(ctx context.Context) (deploy.ManifestList, error) {
 	var manifests deploy.ManifestList
 	for _, kustomizePath := range k.KustomizePaths {
-		cmd := exec.CommandContext(ctx, "kustomize", buildCommandArgs(k.BuildArgs, kustomizePath)...)
-		out, err := util.RunCmdOut(cmd)
+		var out []byte
+		var err error
+
+		if k.useKubectl {
```

Comment:
Worth noting that `kubectl customize` only supports a subset of `kustomize` functionality?

Reply:
yeah, this could manifest through build args being rejected by the binary depending on which version the user *thinks* they're using - but I think that's ok left up to them to handle? we can't really know which version of kustomize a user is targeting with their arguments without asking them, so I think it's ok to assume that whatever they pass will be compatible with whatever kustomize binary we end up using. this is of course assuming that they can easily know which one we'll select based on good documentation :)